src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -21,34 +25,42 @@ import CardSectionDataEmpty from './CardSectionDataEmpty';
 import CardSectionUtils from './utils';
 
 function CardSection() {
+    const [isRequestRefundModalVisible, setIsRequestRefundModalVisible] = useState(false);
     const {translate, preferredLocale} = useLocalize();
     const styles = useThemeStyles();
     const theme = useTheme();
     const [fundList] = useOnyx(ONYXKEYS.FUND_LIST);
+    const subscriptionPlan = useSubscriptionPlan();
+    const isEligibleForRefund = useIsEligibleForRefund();
```

Comment:
I can see we've removed hook but does it make sense to just access the variable here? If you notice we are accessing `const [account]` in the next line.